docs/user-guide/consolidated_metadata.rst

```diff
@@ -0,0 +1,119 @@
+.. only:: doctest
+
+   >>> from pprint import pprint
```

Comment:
I think worth showing this and importing it where it's used instead of hiding it away at the top, so users trying to run the code know where the `pprint` command is coming from?

Reply:
See #2632